README.md

```diff
@@ -29,6 +29,7 @@ Install the package `hub-xmlrpc-api`, available from SUSE Manager 4.1 and Uyuni
  - `HUB_API_URL`: URL to the Hub XMLRPC API endpoint
  - `HUB_CONNECT_TIMEOUT`: maximum number of seconds to wait for a response when connecting to a Server
  - `HUB_REQUEST_TIMEOUT`: maximum numbr of seconds to wait for a response when calling a Server method
+ - `USE_SSL`: use https protocol for communication between Daemon and peripheral servers. Certificates for all the servers will be needed on the machine where Daemon will be running.
```

Comment:
Please expand with detailed steps how to retrieve a certificate from a peripheral Server and install it (assume SLES on the machine running hub-xmlrpc-api).